components/mod-event/EventList.tsx

```diff
@@ -478,29 +478,29 @@ const EventFilterPanel = ({
           <FormLabel
             label="Report Reason"
             htmlFor="reasonType"
-            className="flex-1 max-w-sm"
+            className="mt-2"
           >
-            <Select
-              isMultiple
-              isSearchable
-              primaryColor=""
-              value={reportTypes.map((value) => ({
-                value,
-                label: reasonTypeOptions[value],
-              }))}
-              options={Object.entries(reasonTypeOptions).map(
-                ([value, label]) => ({
-                  label,
-                  value,
-                }),
-              )}
-              onChange={(value) =>
-                changeListFilter({
-                  field: 'reportTypes',
-                  value: Array.isArray(value) ? value.map((v) => v.value) : [],
-                })
-              }
-            />
+            {Object.keys(reasonTypeOptions).map((typeValue) => {
```

Comment:
Move the report type selector from searchable dropdown to clickable buttons just like labels.